process.py

```diff
@@ -399,10 +406,15 @@ def main():
             db.session.query(table).delete()
             db.session.commit()
 
-    for directory in glob.glob("{0}/*/*".format(args.path)):
-        parent_directory = os.path.basename(os.path.dirname(os.path.join(os.pardir, directory)))
-        provider_code = args.provider_code if args.provider_code else get_provider_code(parent_directory)
-        import_to_datastore(directory, provider_code, args.batch_size)
+    if specific_folder == 'y' or specific_folder == 'Y':
```

Comment:
Don't use the same check as before (what if you want to change the condition to also allow 'YES', for example?).
Use a variable to keep the boolean answer if you don't have a better way.